### ISSUE TITLE
redeploy required after initial setup

### ISSUE THREAD

**Issue by pcrockett** [pcrockett/mollysocket-fly#8]
> ...running `mollysocket connection ping {uuid}` works successfully and notifies test message. But actual messages still remain undelivered. 
>
> ...
>
> I have no idea what's causing the issue or how fly.io works but for me, solution was to run `fly deploy` after the issue(or maybe after once mollysocket is fully setup)and it fixed the issue. Working good so far since last 24hours.
>
> Tested this twice. Same issue. Same solution. 
> Hint might be in what's the difference between fly launch and fly deploy.

_Originally posted by @jayb-g in https://github.com/pcrockett/mollysocket-fly/issues/6#issuecomment-2092443173_

**Comment by pcrockett** [comment]
@jayb-g I am still unable to reproduce this "re-deploy" fix personally. However I did discover in MollySocket documentation that battery optimization should be disabled for the _Molly_ app, not just the UnifiedPush distributor app. Is it possible that this could be the real issue here?

**Comment by jayb-g** [comment]
No, I already had battery set to unrestricted for molly. 

> I am still unable to reproduce this "re-deploy" fix personally

That is the only command I ran in both times I tested and it got fixed afterwards. Not sure what other effects that command has.

**Comment by pcrockett** [comment]
> Did setting battery to unrestricted resolve the notifications delay in your case?

It did seem to do the trick for a phone with a different OS than mine (and different user with different usage patterns).

**Comment by jayb-g** [comment]
Will update here if I stumble upon the same issue when I set it up again

**Comment by pcrockett** [comment]
Much appreciated @jayb-g. I added a `logs` target to the Makefile. Next time you add a connection, the output from `make logs` might be very helpful.

**Comment by pcrockett** [comment]
@jayb-g I took a look at MollySocket code and came to the conclusion that you must be right. Running `mollysocket connection add...` does not cause the server to start _monitoring_ the connection. It only adds the connection to the MollySocket database.

Running `flyctl deploy` will restart the server, which then causes it to pick up the new connection.

Added instructions to the readme. Thanks again for your help.
